reference/normalize_test.go

```diff
@@ -677,6 +683,7 @@ func TestParseDockerRef(t *testing.T) {
 		},
 	}
 	for _, test := range testcases {
+		test := test
```

Comment:
Nice catch. I would also love this var to (and the `range` var) to be called `tc` for consistency's sake but I appreciated make tests consistence would've been a proper effort

Reply:
Yes, 💯. Both for the OCD side of me, but also because having a common pattern / naming helps with cognitive overload when reading code (when reading `tc` you don't have to "think" what it is).

Probably best to do so in a one-time pass over all existing tests and update them.